packages/frontend/scss/gallery/_media.scss

```diff
@@ -14,12 +14,28 @@ $tab-height: 46px;
     flex-grow: 1;
     text-align: end;
     align-self: center;
-    padding-right: 12px;
+    margin-right: 12px;
     font-size: 18px;
     color: var(--summary-text-color);
     white-space: nowrap;
   }
 
+  .tab-list {
+    display: flex;
+    justify-content: space-between;
+    .tab-item {
+      margin-right: 30px;
+      font-size: 1.1rem;
+      padding-bottom: 6px;
+      &.selected,
+      &:hover {
+        color: #42a5f5;
+        border-bottom: 3px solid #42a5f5;
+        cursor: pointer;
+      }
+    }
+  }
```

Comment:
The tabs jump in size.

https://github.com/user-attachments/assets/737e2881-885e-4327-a8a5-2effd083182e

Though it is bearable I suppose.

Reply:
I added a flex grow 1 element to avoid layout jump and flex gap to have independant margins between the tab titles